pkg/backup/backup.go

```diff
@@ -444,6 +446,50 @@ func (kb *kubernetesBackupper) BackupWithResolvers(
 	}
 
 	var itemBlock *BackupItemBlock
+	itemBlockReturn := make(chan ItemBlockReturn)
```

Comment:
Should this channel be a buffered channel ?

Reply:
@shubham-pampattiwar Good question. For some reason the example I was looking at earlier didn't do this, but maybe we do want that.